array-loops.js

```diff
@@ -15,6 +21,13 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // --------------------------
 
 const numbers = [2, 4, 5, 29, 38];
```

Comment:
Ook voor deze opdracht geld dat je het jezelf een stuk makkelijker had gemaakt door een for-loop te gebruiken.